spec/models/resource_spec.rb

```diff
@@ -98,7 +98,7 @@
 
   describe '#max_out_attempts' do
     before { @resource.max_out_attempts }
-    its(:attempts) { is_expected.to be >=  Setting['advanced.attempts'].to_i }
+    its(:attempts) { is_expected.to be >=  ENV['attempts'].to_i }
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.